sentry-android-core/src/main/java/io/sentry/android/core/gestures/ViewUtils.java

```diff
@@ -0,0 +1,101 @@
+package io.sentry.android.core.gestures;
+
+import android.content.res.Resources;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AbsListView;
+import android.widget.ScrollView;
+import androidx.core.view.ScrollingView;
+import io.sentry.util.Objects;
+import java.util.ArrayDeque;
+import java.util.Queue;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+final class ViewUtils {
+  static @Nullable View findTarget(
+    final @NotNull View decorView,
+    final float x,
+    final float y,
+    final @NotNull ViewTargetSelector viewTargetSelector
+  ) {
+    Queue<View> queue = new ArrayDeque<>();
+    queue.add(decorView);
+
+    @Nullable View target = null;
```

Comment:
I don't recall us using annotations within a method but it's not a bad idea.

Reply:
yeah I realized it a bit later, that we don't do that, but I guess I will just leave it